packages/phasor/src/elements/brush/brush-element.ts

```diff
@@ -0,0 +1,76 @@
+import { Utils } from '../../utils/tl-utils.js';
+import { Vec } from '../../utils/vec.js';
+import { deserializeXYWH, serializeXYWH } from '../../utils/xywh.js';
+import { BaseElement, HitTestOptions } from '../base-element.js';
+
+export class BrushElement extends BaseElement {
+  type = 'brush' as const;
+  color = '#000000';
+  points: number[][] = [];
+  lineWidth = 4;
+
+  get tiny() {
+    return this.w <= this.lineWidth && this.h <= this.lineWidth;
+  }
+
+  hitTest(x: number, y: number, options?: HitTestOptions) {
+    const point = Vec.sub([x, y], [this.x, this.y]);
+
+    if (this.tiny) {
+      return Utils.pointInCircle(point, [0, 0], this.lineWidth / 2);
+    }
+    return Utils.pointInPolyline(point, this.points, this.lineWidth);
+  }
+
+  render(ctx: CanvasRenderingContext2D): void {
+    ctx.beginPath();
```

Comment:
No matching `closePath`, this will lead to performance degeneration after running for a while.

Reply:
So the better solution is using `Path2D`? I'm not sure and still try to find some information.

Reply:
LGTM